### ISSUE TITLE
Search: Note about the search filters is not placed logically

### ISSUE THREAD

**Issue by sarasuri** [mmiron-HC/WST-RSAMS#160]
Page: Search
URL: https://rsams.dev.cloud.openplus.ca/en/search/site
User Agent: Firefox (v99)
Language: English and French

Issue Description: The note about the search filters is not placed in a logical order. It could be missed by users with visual impairments as it is placed after the "Search" feature.

![image](https://user-images.githubusercontent.com/12256067/162241866-b9c7eef7-14b9-43b0-8c26-32d13bd229d8.png)

Steps to reproduce:

1. Visual scanning

Expected results: The note should be available to users before they encounter the search feature. In this case, it should be placed below the "h1" heading.

**Comment by echouin** [comment]
Text has been removed. Resolved.

**Comment by 40999221** [comment]
The layout has been changed. "Note" has been removed. Issue no longer exist.

![image](https://github.com/user-attachments/assets/174115e2-b413-46e2-8033-d1e48c36848c)

**Comment by zimw7** [comment]
Issue no longer exists. Layout has changed and the note has been removed.
![image](https://github.com/user-attachments/assets/8e80ec77-a6c7-447c-a2e1-512962355d82)

**Comment by YuxuanXie01** [comment]
The issue is no longer present since the note section has been removed.
